superset/views/utils.py

```diff
@@ -101,15 +101,18 @@ def get_viz(
 
 def get_form_data(slice_id=None, use_slice_data=False):
     form_data = {}
-    post_data = request.form.get("form_data")
```

Comment:
@villebro I was spinning my wheels for quite some time trying to get `test_request_context()` to work and then realize that this isn't explicitly related to the POST method and not is the body of the request a JSON object, i.e., the `form_data` field is actually further encoded as a JSON object. 

The form-data is messy and we probably should strive to standardize this at some stage. For now I've renamed `post_data` as `request_form_data` which is more accurate.